apps/console/src/features/applications/components/settings/attribute-management/attribute-selection.tsx

```diff
@@ -468,6 +471,13 @@ export const AttributeSelection: FunctionComponent<AttributeSelectionPropsInterf
                                                     )
                                                 }
                                             </Table.Row>
+                                            <Table.Row>
+                                                <Table.Cell colSpan='2'>
+                                                    <Hint>
+                                                        { t("console:develop.features.applications.edit.sections.attributes.selection.mandatoryAttributeHint") }
```

Comment:
Possible line length exceeding. Stick to under 120 characters per line. Check other places as well.

Reply:
Fixed with `65ed0fc`